### PR TITLE
UI Redesign Implementation

### PR DESCRIPTION
**Pull request by amilajack**
- [x] Updated configs to ERB
- [x] CI Install with Yarn
- [x] Initial basic e2e tests
- [x] Query implementation
- [x] Home page redesign
- [x] Initial export implementation
- [x] Update falcon-core to `v0.0.1-8`
- [x] Fix eslint, flow errors

### CODE REVIEW

File: app/api/Database.js [hunk 1]
```diff
@@ -116,4 +116,19 @@ async function getDatabases(): Promise<Array<DatabaseType>> {
   );
 }
 
+export async function exportJson(
```
Comment (amilajack) [inline app/api/Database.js]: Let's change this to just `export()` that can take a type and a path:

```js
type exportTypes = 'csv' | 'json'

export: (type: exportTypes, absolutePath: string exportOptions)
```

`export()` should return the return value of `connection.exportJson`. Also try using a switch statement to choose which method you call.

Here's an example:
```js
function export(type: exportTypes, absolutePath: string, exportOptions) {
  switch(type) {
    case 'json':
      return connection.exportJson(exportPath, exportOptions);
    case 'csv':
      return connection.exportCsv(exportPath, exportOptions);
    default:
      throw new Error(`Invalid or unsupported export type "${type}"`);
  }
}
```

Comment (jooohhn) [inline app/api/Database.js]: Isn't `export` a reserved word in JS?

Comment (amilajack) [inline app/api/Database.js]: oh lol whoops yup!

Comment (amilajack) [inline app/api/Database.js]: How about `exportDatabase()`

Comment (jooohhn) [inline app/api/Database.js]: Changed it to `exportFile` until I saw your suggestion. You still good with my nomenclature?

**Pull request merged**